common/ActionManager.js

```diff
@@ -25,12 +47,11 @@ class ActionManager {
   defineActions(definitions) {
     Object.keys(definitions).forEach(name => {
       const definition = definitions[name];
-      const composed = function() {
-        return compose([
+      const composed = (...args) =>
```

Comment:
Destructuring arguments was broken in Firefox 45, so this will break anyone on that version (looks like about 2.5% of pings or so, not that many)

(see https://bugzilla.mozilla.org/show_bug.cgi?id=1067049)